pkg/codegen/internal/test/testdata/simple-enum-schema/python/pulumi_plant_provider/_inputs.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+# *** WARNING: this file was generated by test. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import warnings
+import pulumi
+import pulumi.runtime
+from typing import Any, Mapping, Optional, Sequence, Union
+from . import _utilities, _tables
+from . import _enums as enums
```

Comment:
Despite use of `__all__` in this module, should we keep this named `_enums` (with the underscore) as a deterrent to keep it out of intellisense?

```suggestion
from . import _enums
```